global_translations.R

```diff
@@ -38,3 +38,27 @@ complete_trans_df <- subset(final_df, untranslated_count == 0, select = -c(fuzzy
 
 # Untranslated Packages with respective languages associated
 complete_untrans_df <- subset(final_df, translated_count == 0, select = -c(fuzzy_count))
+
+# R Language Translation Team Contact details
+library(rvest)
+library(stringr)
+
+translation_teams <- read_html("https://developer.r-project.org/TranslationTeams.html") |>
+  html_node("table") |>
+  html_table()
+
+members_contact <- translation_teams$Contact |>
```

Comment:
here we create `members_contact` then below assign it back to `translation_teams$Contact`, otherwise `members_contact` is not used. Let's just assign back to the column directly:

```r
translation_teams$Contact <- translation_teams$Contact |> ...
```

Reply:
Yeah. it will also reduce unnecessary variable assignment. Will update that.